src/components/Nurseries/TableCellRenderer.tsx

```diff
@@ -20,7 +20,7 @@ export default function NurseriesCellRenderer(props: RendererProps<TableRowType>
   }
 
   if (column.key === 'timeZone') {
-    return <CellRenderer index={index} column={column} value={value || defaultTimeZone.longName} row={row} />;
```

Comment:
replaced with useGetTimeZone which falls back to default.. also returns the longName (previously we were displaying the id)

Reply:
Oh, thanks for the fix!